cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -224,6 +228,9 @@ public void configure(Map<String, ?> configs) {
       reconnectBackoffMs = String.valueOf(DEFAULT_RECONNECT_BACKOFF_MS);
     }
 
+    _acceptableMetricRecordProduceDelayMs = ACCEPTABLE_NETWORK_DELAY_MS + new CruiseControlMetricsReporterConfig(configs, false)
```

Comment:
Can we also add a config sanity-check to ensure that the sampling rate cannot be more frequent than the `CRUISE_CONTROL_METRICS_REPORTER_INTERVAL_MS_CONFIG `?